ui/src/app/shared/components/repo.tsx

```diff
@@ -4,5 +4,11 @@ import {repoUrl} from './urls';
 export const Repo = ({url, children}: {url: string; children?: React.ReactNode}) => {
```

Comment:
Did you mean to change `repo.tsx` in this PR?

Reply:
Same as above